### DISCUSSION TITLE
I can't connect okx wallet in unity android

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#182] [Q&A]
i try to connect okx wallet in unity.
but just i see this picture.
i want to see next step

![화면 캡처 2024-03-22 205758](https://github.com/WalletConnect/WalletConnectUnity/assets/109068040/42edc814-7cbc-4a5b-8ab8-dda3a83de1e8)

**Comment by skibitsky** [comment]
Please open an issue with "Bug Report" template.

**Reply by matthew2099** [reply]
I see
